tools/operator/testclient/operations/instantiatechaincode.go

```diff
@@ -206,6 +206,17 @@ func (i InstantiateCCUIObject) getEndorsementPolicy(organizations []inputStructs
 func (i InstantiateCCUIObject) getMSPIDForOrg(connProfilePath, orgName string) (string, error) {
 	var config GetMSPID
 	var mspID string
+	if !(strings.HasSuffix(connProfilePath, "yaml") || strings.HasSuffix(connProfilePath, "yml")) {
+		files, err := ioutil.ReadDir(connProfilePath)
+		if err != nil {
+			return "", err
+		}
+		for _, file := range files{
+			if strings.Contains(file.Name(), orgName){
+				connProfilePath = paths.JoinPath(connProfilePath, file.Name())
```

Comment:
I am not sure about your usecase, i.e. what the connprofile filenames look like, but I think this might try to add more than one filename to the end of the existing connProfilePath. If orgname is "org1" (or any other substring of multiple names), and the directory has 12 orgs in it, such as org1, org2, org3,...,org10, org11, org12, then this would append: connProfilePath/org1org10org11org12. Can we expect the orgname to be exact and check for a match instead of "Contains"? And/Or should we break out when one connProfiePath is found?